### DISCUSSION TITLE
Feature Request: Add support for loading fonts from in-memory bytes in ttLib.TTFont

### DISCUSSION THREAD

**Discussion by cocoyamnut** [fonttools/fonttools#3577] [Q&A] [answered]
Currently, ttLib.TTFont only supports loading fonts from file paths or file-like objects. However, there are situations where the font data is already loaded into memory as bytes, and it would be more efficient to load the font directly from memory instead of writing it to a temporary file.
This issue proposes adding a new method or modifying the existing constructor to support loading fonts directly from in-memory bytes.

**Comment by justvanrossum** [comment, marked as answer]
You can wrap your bytes in an `io.BytesIO` object and you have the behavior you want.
